vuu-ui/packages/vuu-filters/src/filter-toolbar/useFilterToolbar.tsx

```diff
@@ -35,15 +31,16 @@ const filterToControl = (
       >
         <FilterDropdown
           column={column}
-          selected={value}
+          selected={value.toString()}
```

Comment:
This is ok . I think to fully capture the data type FilterDropdown should maybe be generic, just as the nested Dropdown is (defaults to Dropdown<string>. Gets complicated though given that this is highly dynamic and may be a number or string. toString on everything is the most straightforward option